### DISCUSSION TITLE
Quarkus 3.17.5 - Maintenance release

### DISCUSSION THREAD

**Discussion by giscus** [quarkusio/quarkus#45226] [Quarkus Blog/Website]
# Quarkus 3.17.5 - Maintenance release

Quarkus: Supersonic Subatomic Java

https://quarkus.io/blog/quarkus-3-17-5-released/

**Comment by doogle-oss** [comment]
As a Quarkus user for over 3 years, its very difficult to convince people in large organization to use it. In my org everyone uses Spring Boot. I ran 2 program with over 18 services with Quarkus but apparently once I moved out the program was rewritten with Spring boot again. 

Its good to see great books like quarkus in action. Showcasing the ease to use the framework. But more sales pitch is needed for wider adoption.
Took over a week to go through the book and also rewrite to run with Quarkus 3.17.5. It works wonders. Thanks for the effort to bring this on. I used a lot of external services for the apps to run. It was fun to learn GRPC which was long due for me. Thanks for sharing and writing the book. Great details and explanations.
My code is at 
https://github.com/debu999/quarkus-in-action

I used the following for services

Dev Services
1. MySQL
2. KeyCloak

Docker Local
1. Prometheus
2. Jaeger
3. Redpanda

External Services
1. Mongodb - Mongodb Atlas free database hosting
2. Postgres - Neon.tech free postgres hosting
3. Grafana - Free at graphana.net
4. RabbitMQ - Free hosting at cloudamqp.com

#Quarkus #Redhat #GRPC #2025learning
